.github/workflows/code_scanning.yaml

```diff
@@ -15,17 +15,7 @@
 name: "CodeQL"
 
 on:
-  pull_request:
-    types:
-      - opened
-      - synchronize
-    branches:
-      - main
-      - release-*
```

Comment:
Is there any reason to not use the "standard" triggers for this?

Reply:
I think is a good first step to run the code scan before performing any extra steps.

Reply:
That wasn't my question. Does it make sense to *also* run this on PRs?